tests/Unit/FormRequests/AdminNewUpdateTraderRequestTest.php

```diff
@@ -76,6 +76,38 @@ public function storeValidationProvider()
                     'market' => 1,
                 ]
             ],
+            'requestShouldSucceedWhenOptionalProvidedAsBooleanInt' => [
+                'passed' => true,
+                'data' => [
+                    'name' => 'Test Trader',
+                    'market' => 1,
+                    'disabled' => 1,
+                ]
+            ],
+            'requestShouldSucceedWhenOptionalDataProvidedAsNull' => [
+                'passed' => true,
+                'data' => [
+                    'name' => 'Test Trader',
+                    'market' => 1,
+                    'disabled' => null,
+                ]
+            ],
+            'requestShouldSucceedWhenOptionalDataProvidedAsBoolean' => [
+                'passed' => true,
+                'data' => [
+                    'name' => 'Test Trader',
+                    'market' => 1,
+                    'disabled' => true,
+                ]
+            ],
+            'requestShouldSucceedWhenOptionalDataProvidedAsBooleanText' => [
+                'passed' => true,
+                'data' => [
+                    'name' => 'Test Trader',
+                    'market' => 1,
+                    'disabled' => "on",
```

Comment:
Why do you have to allow this to be "on", true or 1 ?

Reply:
those are the affirmative bits of data the validation rule will permit.